### DISCUSSION TITLE
EPPROM Backup Issue

### DISCUSSION THREAD

**Discussion by FRPProject** [OpenI6X/opentx#125] [Q&A] [answered]
I have been trying to restore my EPPROM backup from OpenI6x 1.1.1 to the latest version 1.2.0. But, I have not been successful. I have done it before on previous version with success following https://youtu.be/Z5zY_yA0avw, but on the latest version the device formats the EPPROM after restoring. How can I resolve this issue? Thank you.

**Comment by ajjjjjjjj** [comment, marked as answer]
You have to create your models manually, eeprom settings version has changed and cannot be restored from previous versions.
